src/gui/mainwindow.cpp

```diff
@@ -427,6 +428,7 @@ MainWindow::MainWindow(QWidget *parent)
 #endif
 
     m_propertiesWidget->readSettings();
+    m_ui->actionShowFiltersSidebar->setChecked(isFiltersSidebarVisible());
```

Comment:
Have you tried it in a clean installation? I mean when there are no config file and does it still work correctly?